src/components/Rapidoc.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Helmet from 'react-helmet'
+
+const Rapidoc = () => {
+  return (
+    <div className='container docs-page'>
+      <Helmet>
+        <script type="module" src="https://unpkg.com/rapidoc/dist/rapidoc-min.js"></script>
```

Comment:
I need to know where/how to put this `.js` in the tree.

Drop it in static and commit? Some yarn/gatsby thing I do not know?

Reply:
Will also defer to others here, but could you install the rapidoc dep through yarn and import it into this React component? I think I found a good example [here](https://codesandbox.io/s/upbeat-sea-4tfo7?file=/src/App.js)

Reply:
I pulled `https://unpkg.com/rapidoc/dist/rapidoc-min.js` into `/static/js`.

Reply:
Sorry, your comment didn’t display before I replied to myself.

No idea. Looks like the right way to do it, I’ll give it a shot.